src/components/FiltersBar.js

```diff
@@ -0,0 +1,131 @@
+import { useProduct } from "../context/product-context";
+import { GetFeaturedCategories } from "../utils/getFeaturedCategories";
+
+
+export const FiltersBar = ()=>{
+  const {state,dispatch} = useProduct();
```

Comment:
suggestion: try to be more descriptive with the names, you can name it like `filterState` , `dispatchFilters`

Reply:
Done changes in new commit.